docs/project-model/project-json-reference.md

```diff
@@ -0,0 +1,494 @@
+Type: String
+
+The version of the language used by the compiler.
+
+<a name="allowUnsafe"></a>
+### allowUnsafe
+Type: Boolean
+
+Allows unsafe code in this project.
+
+<a name="platform"></a>
+### platform
+Type: String
+
+The name of the target platform, such as AnyCpu, x86 or x64.
+
+<a name="warningsAsErrors"></a>
+### warningsAsErrors
+Type: Boolean
+
+If set to true, treats warnings as errors.
+
+<a name="optimize"></a>
+### optimize
+Type: Boolean
+
+Enables the compiler to optimize the code in this project.
+
+<a name="keyFile"></a>
+### keyFile
+Type: String
+
+The path for the key file used for signing this assembly.
+
+<a name="delaySign"></a>
+### delaySign
+Type: Boolean
+
+Causes signing to be delayed.
+
+<a name="publicSign"></a>
+### publicSign
+Type: Boolean
+
+Enables signing of the resulting assembly.
+
+<a name="emitEntryPoint"></a>
+### emitEntryPoint
+Type: Boolean
+
+When set to true, the project will compile to an exe file. if se tto false, the project will produce a dll.
+
+<a name="xmlDoc"></a>
+### xmlDoc
+Type: Boolean
+
+Enables xml docs to be generated from comments in the sourcecode.
+
+<a name="preserveCompilationContext"></a>
+### preserveCompilationContext
+Type: Boolean
+
+(tbd)
+
+<a name="analyzerOptions"></a>
+## analyzerOptions
+Type: Object
+
+An object with properties used by code analysers.
+
+<a name="languageId"></a>
+### languageId
+Type: String
+
+The id of the language to analyze.
+
+    "analyzerOptions": {
+      "languageId": "vb"
+      }
+    }
+
+<a name="configurations"></a>
+## configurations
+Type: Object
+
+An object whos properties define different configurations for this project, such as Debug and Release. Each value is an object that can contain a compilationoptions object with options specific for this coniguration.
+
+    "configurations": {
+      "Release": {
+        "compilationOptions": {
+          "allowUnsafe": false
+        }
+      }
+    }
+
+<a name="frameworks"></a>
+## frameworks
+Type: Object
+
+An object whose properties specifies what frameworks this project supports, such as .NetFramework or UWP. Must be a valid target framework moniker (TFM). Each value is an object that can contain information specific to this framework such as compilationoptions, analyzeroptions as well as the following properties.
+
+    "dependencies": {
+        "Microsoft.Extensions.JsonParser.Sources": {
+            "type": "build",
+            "version": "1.0.0-rc2-16453"
+        },
+        "Microsoft.Extensions.HashCodeCombiner.Sources": {
+            "type": "build",
+            "version": "1.0.0-rc2-16054"
+        },
+        "Microsoft.Extensions.DependencyModel": "1.0.0-*"
+    },
+
+<a name="dependencies"></a>
+### dependencies
+Type: Object
+
+Dependencies that are specific for this framework. Useful if for example one framework has built in support for something that another framework doesnt, or if a diffrent version of a library is required for a perticular framework.
```

Comment:
I'd change the second sentence to something like:

"This is useful in scenarios where you cannot simply specify a package-level dependency across all targets.  Reasons for this can include one target lacking built-in support that other targets have, or requiring a different version of a dependency than other targets."